README.md

```diff
@@ -33,3 +33,13 @@ Prospective Collaborators:
 - https://github.com/codeisland/makermap
 
 To join us, please email our team at anansegroup@gmail.com
+
+Steps to get ready with the project
```

Comment:
Use markdown syntax to separate the sections with headers. Also, please make the same with `General Details`, `Collaborators`, and `Prospective Collaborators`.